iceoryx_utils/cmake/IceoryxPlatformDetection.cmake

```diff
@@ -17,11 +17,11 @@ elseif(APPLE)
 endif(LINUX)
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-    set(TEST_CXX_FLAGS ${ICEORYX_WARNINGS} /W1)
+    set(ICEORYX_WARNINGS PRIVATE ${ICEORYX_WARNINGS} /W1)
```

Comment:
this is weird, why is this shown here as change?

Reply:
I guess something went wrong with Git. Can you rebase and squash again @elfenpiff?